Description: Fix uninitialized variable warning
 If the YUV format isn't found, the code depends on .format being
 0. However, it was not initialized at all, so this patch initializes
 the format member. The other members need not be initialized.
Author: Jiri Palecek <jpalecek@web.de>
Last-Update: 2020-06-04
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: kwin-5.18.5/platformsupport/scenes/opengl/egl_dmabuf.cpp
===================================================================
--- kwin-5.18.5.orig/platformsupport/scenes/opengl/egl_dmabuf.cpp
+++ kwin-5.18.5/platformsupport/scenes/opengl/egl_dmabuf.cpp
@@ -321,7 +321,7 @@ KWayland::Server::LinuxDmabufUnstableV1B
                                                                     const QSize &size,
                                                                     Flags flags)
 {
-    YuvFormat yuvFormat;
+    YuvFormat yuvFormat { 0 };
     for (YuvFormat f : yuvFormats) {
         if (f.format == format) {
             yuvFormat = f;
